sippy-ng/src/component_readiness/CompReadyTestReport.js

```diff
@@ -125,8 +122,28 @@ export default function CompReadyTestReport(props) {
         }
       })
       .finally(() => {
-        // Mark the attempt as finished whether successful or not.
-        setIsLoaded(true)
+        isReportDone = true
+        checkFetchesDone()
+      })
+
+    fetch(process.env.REACT_APP_API_URL + '/api/releases')
+      .then((response) => response.json())
+      .then((data) => {
+        let tmpRelease = {}
+        data.releases
+          .filter((aVersion) => {
+            // We won't process Presubmits or 3.11
+            return aVersion !== 'Presubmits' && aVersion != '3.11'
+          })
+          .forEach((r) => {
+            tmpRelease[r] = data.ga_dates[r]
+          })
+        setVersions(tmpRelease)
+      })
+      .catch((error) => console.error(error))
+      .finally(() => {
+        isVersionFetchDone = true
+        checkFetchesDone()
```

Comment:
We're duplicating this in a few places, I think it'd be better to have a context for the release API.  It's the solution I'm working on as an alternative to #1058.